DQM/TrackingMonitor/src/TrackSplittingMonitor.cc

```diff
@@ -148,15 +156,6 @@ void TrackSplittingMonitor::bookHistograms(DQMStore::IBooker & ibooker,
 void TrackSplittingMonitor::beginJob(void) 
```

Comment:
I suggest to remove beginJob and ednJob methods completely: they are bogus for this module type (framework will never call them, either single thread or multithread). Also, add ``` override``` in the header file to the virtual derived methods.